service/src/main/java/org/sunbird/actor/systemsettings/SystemSettingsActor.java

```diff
@@ -52,4 +56,14 @@ private void setSystemSetting(Request actorMessage) {
     Response response = service.setSystemSettings(request, actorMessage.getRequestContext());
     sender().tell(response, self());
   }
+
+  private void getSystemSettingV2(Request actorMessage) {
+    JsonNode setting =
+        service.getSystemSettingV2ByKey(
+            (String) actorMessage.getContext().get(JsonKey.FIELD),
+            actorMessage.getRequestContext());
+    Response response = new Response();
+    response.put(JsonKey.RESPONSE, setting);
```

Comment:
there is a chance that null is returned for setting variable... in that case we return 200 OK with null value ??
check this out and make sure we return 404 error when the key is not found with proper json response body with proper errmsg